modules/ROOT/pages/math.adoc

```diff
@@ -0,0 +1,129 @@
+= Math plugin
+:navtitle: Math Plugin
+:description: {pluginname} plugin provides a way for users to insert math equation into the {productname} editor.
+:description_short: Insert `+<math>+` elements into {productname}.
+:keywords: plugin, math, formulas
+:pluginname: Math
+:plugincode: math
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.1v.adoc[]
+
+The {pluginname} plugin provides a way for users to insert complex formulas.
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+== Basic setup
+
+To setup the {pluginname} plugin user-interface in the editor:
+
+* add `+math+` to the plugins option in the editor configuration;
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'math',
+  toolbar: 'math',
+});
+----
+
+== How to Use
+
+To work with math elements, follow these steps:
+
+. Open the {productname} editor with the {pluginname} plugin enabled.
+.. Click the {pluginname} plugin icon in the toolbar to `insert` or `edit` a math element.
+.. This opens the {pluginname} dialog which allows the user to choose between either `LaTeX` and `MathML` equations for the formulas field.
+.. By default, `LaTeX` is selected.
+. In the {pluginname} dialog, use the textarea when inserting formulas.
+.. Choose to `insert` the formula either as a **block** paragraph or **inline** with text.
+. Once the formula has been added, the preview section within the modal dialog will render the final math equation.
+. After reviewing the formula, click the `save` button.
+.. This inserts the math equation into the editor, wrapped in either a `+<tiny-math-block>+` or `+<tiny-math-inline>+` element based on the **Text Wrap** chosen.
+
+[TIP]
+The formula is inserted at the last cursor position before the modal dialog was opened.
+
+=== Inserting LaTeX formulas (Block)
```

Comment:
Do we need to have separate headings for both Latex and MathML or can we just have one math block heading and one math inline heading? They result in essentially the same HTML except for the `formula` attribute.